portal/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -0,0 +1,73 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dao.repository.ProjectRepository
+import cromwell.pipeline.datastorage.dto.{ Project, ProjectAdditionRequest, ProjectId, UserId }
+import org.scalamock.scalatest.MockFactory
```

Comment:
We don't use scalamock anymore, we use `Mockito`.

Reply:
done